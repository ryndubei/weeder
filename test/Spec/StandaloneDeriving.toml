roots = [ "Spec.StandaloneDeriving.StandaloneDeriving.root" ]

type-class-roots = false
