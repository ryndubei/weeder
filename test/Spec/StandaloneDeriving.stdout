test/Spec/StandaloneDeriving/StandaloneDeriving.hs:4: A
test/Spec/StandaloneDeriving/StandaloneDeriving.hs:6: (Instance) :: Show A
